harvard-leeds-beckett-university.csl

```diff
@@ -474,9 +474,7 @@
                 </if>
               </choose>
               <group delimiter=" ">
-                <names variable="author">
-                  <name initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-last="never" and="symbol"/>
-                </names>
+                <names variable="author"/>
```

Comment:
how about if there are multiple performers? There's not "and" between them, just commas?